ui/src/app/shared/components/dropdown/dropdown.tsx

```diff
@@ -1,13 +1,14 @@
 import classNames from 'classnames';
-import * as React from 'react';
-import * as ReactDOM from 'react-dom';
+import React, {ComponentType, ReactNode, useEffect, useRef} from 'react';
+import {useState} from 'react';
```

Comment:
```suggestion
import React, {ComponentType, ReactNode, useEffect, useRef, useState} from 'react';
```
consolidate the two import statements together